ffi/blitbuffer.lua

```diff
@@ -1912,4 +1911,28 @@ BB.TYPE_BBRGB16 = TYPE_BBRGB16
 BB.TYPE_BBRGB24 = TYPE_BBRGB24
 BB.TYPE_BBRGB32 = TYPE_BBRGB32
 
+BB.has_cblitbuffer = false
+if not os.getenv("KO_NO_CBB") then
+    -- Load C blit buffer, we'll decide whether to use it later on
+    BB.has_cblitbuffer, cblitbuffer = pcall(ffi.load, "blitbuffer")
+end
+
+-- Set the actual enable/disable CBB flag. Returns the flag of whether it is (actually) enabled.
+function BB:enableCBB(ison)
```

Comment:
For legibility, our variable equivalent of isOn (function) is is_on. ^_^
```suggestion
function BB:enableCBB(is_on)
```

Reply:
But actually is_on seems a bit of a weird name here.

Reply:
I think we generally use:
```suggestion
function BB:enableCBB(enabled)
```
(feel free to suggest alternatives of course, but in that case we should probably update some others too)